client/hotrod-client/src/test/java/org/infinispan/client/hotrod/stress/ClusterClientEventStressTest.java

```diff
@@ -131,8 +133,8 @@ public void testAddClientListenerDuringOperations() {
 //      log.debugf("Put operations completed, assert statistics");
 //      assertStatsAfter(remotecms);
 
-//      log.debugf("Stats asserted, wait for events...");
-//      eventuallyEquals(NUM_EVENTS, () -> countEvents(listeners));
+      log.debugf("Stats asserted, wait for events...");
```

Comment:
This debug statement doesn't make sense if we're not going to do the assertion.

Reply:
The assertion is just below it

Reply:
I mean that the stats haven't been asserted as the line `assertStatsAfter(remotecms);` is commented.

Reply:
I think the idea of the test was to uncomment stuff as needed - however imo if we don't at a minimum assert the events then what is the real point of the test. I can uncomment the above lines if that is what you wanted?

Reply:
So the assert fails. So I think I am going to leave it as is for now :) It also fails on master in the same way, so it isn't caused by this PR.

Reply:
"as needed" doesn't really fit here, because you're unlikely to need an additional assert in a test :)

Too bad the asserts were commented from the beginning so we don't know if they're commented because of a real bug or because the test has wrong expectations /cc @galderz